src/journal-remote/log-generator.py

```diff
@@ -32,7 +32,6 @@
 DATA={data}
 """
 
-m = 0x198603b12d7
 realtime_ts = 1404101101501873
```

Comment:
These 3 other variables are used in the same way as `m`, is there a way to give them the same treatment? Otherwise it looks a bit off